tests/business.test.ts

```diff
@@ -40,23 +22,74 @@ import { Marqeta } from '../src';
       home: {
         address1: '106 Main St.',
         address2: '',
-        city: 'Atlanta',
-        state: 'GA',
-        postalCode: '30345',
+        city: 'Opelousas',
+        state: 'LA',
+        postalCode: '70570',
         country: 'USA'
       }
     },
     identifications: [
       {
         type: 'BUSINESS_TAX_ID',
-        value: '921225677'
+        value: '901721634'
       }
     ],
-  })
-  if (twoB?.success && twoB.body?.token) {
-    console.log(`Success! The Business account ${twoB?.body?.businessNameLegal} was created with token: ${twoB?.body?.token}`)
+  }
+
+  console.log('getting list of Businesses...')
+  const list = await client.business.list()
+
+  if (list?.success && list?.body?.isMore) {
+    console.log('Success! ' + list.body!.count + ' Businesses were retrieved.')
+    const lstItem1 = list?.body?.data?.pop()
+
+    if (lstItem1?.token) {
+      console.log('getting Business account by id: ' + lstItem1.token)
```

Comment:
Again, this is OK, and it works, but it might be easier to read as:

```typescript
     console.log(`getting Business account by id: ${lstItem1.token}`)
```